src/osd/osd_types.h

```diff
@@ -1310,6 +1326,7 @@ struct pg_pool_t {
   __u8 object_hash;         ///< hash mapping object name to ps
 private:
   __u32 pg_num, pgp_num;    ///< number of pgs
+  __u32 pg_num_pending;     ///< pg_num we are about to merge down to
```

Comment:
Identifiers beginning with two underscore are reserved for implementations. Use one underscore, or (my preference) move the underscores to the end of the identifier. (See 2.10.)